ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -91,25 +91,6 @@ StackLayout {
                     }
                 }
             }
-
-            CommunityIntroDialog {
```

Comment:
This one is a duplicate; already present and used as a `Component`

Reply:
I've introduced this one here in `ChatLayout` so that it's available to be spun up by `ChatView` or `JoinCommunityView`.
We gotta make sure that this doesn't break with this removal.

Just raising this here...

Reply:
I just double checked, so:

- leaving a random community from the primary nav: 
![image](https://github.com/status-im/status-desktop/assets/5377645/8f811f40-8141-413e-8a46-d2c7a3a9b984)

- doing the same from Settings/Communities:
![image](https://github.com/status-im/status-desktop/assets/5377645/0a54538f-a6bb-4a53-bcf0-e534c20dc0a6)

Reply:
These are not the intro dialogs though. 
The intro dialogs are shown when you want to request access to a community, right before you enter your password to reveal your addresses.

Reply:
Yeah it's the same dialog, really :) Just has a different set of footer buttons based on the params you set. But the point was that the instance I removed was not being used at all in the existing `ChatLayout.qml`. 

There is:
```qml
Component {
        id: communityIntroDialogPopup
        CommunityIntroDialog {
            id: communityIntroDialog

            property string communityId
...
}
```
which is used as:
```qml
            onRevealAddressClicked: {
                Global.openPopup(communityIntroDialogPopup, {
                    communityId: communityData.id,
                    isInvitationPending: joinCommunityView.isInvitationPending,
                    name: communityData.name,
                    introMessage: communityData.introMessage,
                    imageSrc: communityData.image,
                    accessType: communityData.access
                })
            }
```

Reply:
Ah yes, it's this dialog, still working:

![image](https://github.com/status-im/status-desktop/assets/5377645/224b8f6b-0111-4ac6-ab1c-b2790c9f0a91)

and

![image](https://github.com/status-im/status-desktop/assets/5377645/9ac9e5cf-5be4-40aa-a8f1-5fd1d9eb43fc)